test/dlc_tests/conftest.py

```diff
@@ -276,6 +277,37 @@ def example_only():
     pass
 
 
+@pytest.fixture(scope="session")
+def tf2_only():
+    pass
+
+
+@pytest.fixture(scope="session")
+def tf23_and_above_only():
+    pass
+
+
+@pytest.fixture(scope="session")
+def tf21_and_above_only():
+    pass
+
+
+def tf_version_within_limit(metafunc_obj, image):
+    """
+    Test all pytest fixtures for TensorFlow version limits, and return True if all requirements are satisfied
+
+    :param metafunc_obj: pytest metafunc object from which fixture names used by test function will be obtained
+    :param image: Image URI for which the validation must be performed
+    :return: True if all validation succeeds, else False
+    """
+    tf2_requirement_failed = "tf2_only" in metafunc_obj.fixturenames and is_tf1(image)
```

Comment:
nit: wouldn't it be more appropriate for this condition to be `"tf2_only" in metafunc_obj.fixturenames and not is_tf2(image)` - i.e. it's possible someone uses a fixture other than tensorflow_training or tensorflow_inference but also has this tf2_only fixture.

Reply:
Sure. Alright.

Reply:
Sure. That's true. I'll make the change.